h/views/account_signup.py

```diff
@@ -163,9 +171,27 @@ class IDInfo:
     sub: str
     """The user's unique ID from the third-party provider."""
 
+    email: str | None = None
+    name: str | None = None
+    given_name: str | None = None
+    family_name: str | None = None
```

Comment:
Adding the email and name-related fields to the "idinfo" JWT that we pass to the `/signup/<PROVIDER>` view in a query param.